src/sentry/sentry_metrics/configuration.py

```diff
@@ -10,13 +10,18 @@ class UseCaseKey(Enum):
     PERFORMANCE = "performance"
 
 
+class RateLimiterNamespace(Enum):
+    RELEASE_HEALTH = "releasehealth"
+    PERFORMANCE = "performance"
```

Comment:
Shouldn't this be "pg-releasehealth", "pg-performance" or whatever instead of just mirroring the use case id? Also I don't think should have an Enum here, I'd just make it a string. Isn't it only being used to determine the keyspace? Also it means that ratelimiter has to import this from configuration.py which I think is cleaner to avoid.

Reply:
Ah, yeah keeping the names for backwards compatibility makes sense to me. Wrote this in another comment but I think we should merge the rate limiter cluster options into the same config and not have this enum (and not do that lookup either).